### DISCUSSION TITLE
change the opportunity state to "closed"

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#15726] [How to] [answered]
Hi,

Recently i imported records to opportunity module. Most of the old records status are in the "Sale Completed / Sale failed / Sale cancelled" state. All the imported record status was in "Read only state". I am unable to change the status field. When i select the account name, all the opportunities are in the open state. 

When i create new record in the opportunity field, i can able to select the status as "Sale complete" and its displayed as closed opportunity. Also i can able to change the status field. 

I am facing the issue only for the imported records. Could you please guide to set the imported records to closed state? 

Thanks & regards,
Rajasekaran.

**Comment by rskrzypczak** [comment, marked as answer]
Probably these "Sale Completed / Sale failed / Sale canceled" values do not exist in the dictionary, so you are not able to edit them.
You can try to add these values temporarily in the panel (_Software configuration/Standard modules/Fields - Picklists_), then remove them and convert to the correct labels.
![image](https://user-images.githubusercontent.com/10221999/154626824-0fa7168a-67b0-4bd1-9472-7651fb21fbdf.png)

**Comment by Ilamparithi-Tamizhanangu** [comment]
Hi,

The pickup list already contains the values like "Sale completed / Sale cancelled / Sale failed". 
![image](https://user-images.githubusercontent.com/96602176/154649909-48c33b10-f5ad-4637-8752-0b19da7a0a22.png)

Also it shows read only field for status in imported records only.
![image](https://user-images.githubusercontent.com/96602176/154650022-9b02a0a3-de8c-47f3-835d-b6f1b44ee06a.png)

I can able to edit the manually created fields. I set the status as Sale completed and later  i can change. 
![image](https://user-images.githubusercontent.com/96602176/154650198-6a8c7f29-7068-4a5e-9b87-950813c859f4.png)

If imported closed records are in readonly mode, we doesnt have any issue. they are closed records. But all the records as displayed in the open state for the opportunities.
![image](https://user-images.githubusercontent.com/96602176/154651055-7e510705-72ca-4db2-9f2e-1dd74e587100.png)

Only manually changed to "sale completed" records only displayed as closed opportunities in the accounts page.
![image](https://user-images.githubusercontent.com/96602176/154650674-0c69a8c5-7dd1-4f09-ad32-e47511df7da0.png)


Could you please suggest the solution to convert the imported records to closed opportunities in the accounts page.? 
or

Could you please tell when an opportunity will be marked as closed in the accounts page ? 

Thanks & regards,
Rajasekaran.

**Reply by rskrzypczak** [reply]
Like I said, most likely you used a translated value instead of the label in import.
You can solve this problem by modifying the entries directly in the database, just change 'Sale Completed' => 'PLL_SALE_COMPLETED' etc.
Or you can do it from the GUI like i mentioned previously.

**Reply by Ilamparithi-Tamizhanangu** [reply]
Hi,

I created dummy status in the records (like SaleCOM for Sale Completed) & pickup list. The records are in the edit mode for the status other than Sale Completed, Sale cancelled or Sale failed. 

Now filtered the SaleCOM records and with the help of mass edit, changed to Sale completed. Simillary changed for sale failed and sale cancelled. Since now values are changed within yetiforce, all the records are editable. 

Now Sale completed, sale failed and sale cancelled records are properly displayed as closed opportunities in the accounts page. 

Thanks for your support & issue was solved. 

Thanks & Regards,
Rajasekaran.
